src/components/registerV2/chooseAvatar.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { translate } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import { generatePassphrase } from '../../utils/passphrase';
+import { extractAddress } from '../../utils/account';
+import routes from '../../constants/routes';
+import { FontIcon } from '../fontIcon';
+import { PrimaryButtonV2, SecondaryButtonV2 } from '../toolbox/buttons/button';
+import AccountVisual from '../accountVisual';
+import registerStyles from './registerV2.css';
+import styles from './chooseAvatar.css';
+import avatar from '../../assets/images/icons-v2/avatar.svg';
+
+class ChooseAvatar extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      addresses: [],
+    };
+
+    this.getAvatarAnimationClassName = this.getAvatarAnimationClassName.bind(this);
+  }
+
+  componentDidMount() {
+    /* istanbul ignore next */
+    const crypotObj = window.crypto || window.msCrypto;
+    const passphrases = [...Array(5)].map(() =>
+      generatePassphrase({
+        seed: [...crypotObj.getRandomValues(new Uint16Array(16))].map(x => (`00${(x % 256).toString(16)}`).slice(-2)),
+      }));
+    const addresses = passphrases.map(pass => extractAddress(pass));
+    this.setState({
+      addresses,
+    });
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  getAvatarAnimationClassName(address, selected, previousSelected) {
+    return selected === address
+      ? styles.selected
+      : (previousSelected === address && styles.unselected) || '';
```

Comment:
Is the `previousSelected` really needed, couldn't it be done with css `transition` instead of `animation`? 
It would be great if the `previousSelected` doesn't have to me stored in the react state just for the css.